xclim/core/units.py

```diff
@@ -518,19 +537,24 @@ def to_agg_units(
     >>> degdays = dt.clip(0).sum("time")  # Integral of temperature above a threshold
     >>> degdays = to_agg_units(degdays, dt, op="integral")
     >>> degdays.units
-    'week delta_degC'
+    'K week'
 
     Which we can always convert to the more common "K days":
 
     >>> degdays = convert_units_to(degdays, "K days")
     >>> degdays.units
     'K d'
     """
-    if op in ["amin", "min", "amax", "max", "mean", "std", "sum"]:
+    if op in ["amin", "min", "amax", "max", "mean", "sum"]:
         out.attrs["units"] = orig.attrs["units"]
 
+    elif op in ["std"]:
+        out.attrs["units"] = ensure_absolute_temperature(orig.attrs["units"])
```

Comment:
According to a recent change in CF (CF 1.11) you should also add an attribute `units_metadata="temperature: difference";` if you know that you are witting a difference of temperatures
See: 
http://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#use-of-units-metadata-ex

Reply:
@aulemahal Do you want to address that here, or in another PR?

Reply:
I think this goes in another PR.  OR rather it would be an alternative to this PR. Here we avoid the case `units_metadata="tempature: difference"` by converting to the absolute unit.

Reply:
@bzah Would you mind opening a more elaborate issue for this particular feature?

Reply:
Oups, I missed the replies, sorry!

@aulemahal Aren't you loosing information when converting from delta_degC to K ? 
Just looking at the units K, you can't tell if the data is on scale or a difference.

I think it would be complementary to this PR to add the `units_metadata="temperature: difference";` attribute.
Because then, you have both the workaround you wanted for pint delta units that are not understood by CF and also you have standardized metadata for people how would use the product itself.

Reply:
I guess you're right. From my reading, I thought the `units_metadata` attribute only applied to offset units like degC, not to Kelvins. 
In most of the cases, though, I feel a human would infer the information from context : from the long name and other metadata fields. My fix was for the machines mainly.